crates/node/core/src/args/metric.rs

```diff
@@ -10,4 +10,22 @@ pub struct MetricArgs {
     /// The metrics will be served at the given interface and port.
     #[arg(long="metrics", alias = "metrics.prometheus", value_name = "PROMETHEUS", value_parser = parse_socket_address, help_heading = "Metrics")]
     pub prometheus: Option<SocketAddr>,
+
+    /// URL for pushing Prometheus metrics to a push gateway.
+    ///
+    /// If set, the node will periodically push metrics to the specified push gateway URL.
+    #[arg(long = "metrics.push.url", value_name = "PUSH_GATEWAY_URL", help_heading = "Metrics")]
+    pub push_gateway_url: Option<String>,
+
+    /// Interval in seconds for pushing metrics to push gateway.
+    ///
+    /// Default: 5 seconds
+    #[arg(
+        long = "metrics.push.interval",
+        default_value = "5",
+        value_parser = parse_duration_from_secs,
+        value_name = "SECONDS",
+        help_heading = "Metrics"
+    )]
+    pub push_gateway_interval: Duration,
```

Comment:
one more thing: we may want to add pushing of OTLP traces in the future, so let's change the CLI argument names here
```suggestion
    #[arg(long = "metrics.prometheus.push.url", value_name = "PUSH_GATEWAY_URL", help_heading = "Metrics")]
    pub push_gateway_url: Option<String>,

    /// Interval in seconds for pushing metrics to push gateway.
    ///
    /// Default: 5 seconds
    #[arg(
        long = "metrics.prometheus.push.interval",
        default_value = "5",
        value_parser = parse_duration_from_secs,
        value_name = "SECONDS",
        help_heading = "Metrics"
    )]
    pub push_gateway_interval: Duration,
```